string name
inhus/Action[] actions